readme.md

```diff
@@ -30,6 +30,8 @@ pod 'MapboxDirections.swift', :git => 'https://github.com/mapbox/MapboxDirection
 pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :tag => 'v0.0.4'
 ```
 
+Then add `MapboxNavigation.framework` under `Embedded Binaries` on the `General tab`.
```

Comment:
CocoaPods should do this automatically for the developer, so no additional installation steps are needed. If I remember correctly, the example applications in this workspace is a special case. You could compare this project with MapboxGeocoder.swift to be sure, though.